packages/keyboard-shortcuts/src/store/selectors.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Shared reference to an empty array for cases where it is important to avoid
+ * returning a new array reference on every invocation.
+ *
+ * @type {Array}
+ */
+const EMPTY_ARRAY = [];
+
+/**
+ * Returns the main key combination for a given shortcut name.
+ *
+ * @param {Object} state Global state.
+ * @param {string} name  Shortcut name.
+ *
+ * @return {Object?} Key combination.
```

Comment:
It's referenced by the singular "key" combination here, but the function is named "keys" combination.

Which should be the canonical form? Whichever we choose, can we update the other to be consistent?

Reply:
I'm going with the singular as it seems we already had that before.